docs/docusaurus/sidebars.js

```diff
@@ -198,6 +198,11 @@ module.exports = {
               label: 'Edit a Checkpoint configuration',
               href: '/docs/cloud/checkpoints/manage_checkpoints#edit-a-checkpoint-configuration',
             },
+            {
+              "type": "link",
+              "label": "Configure the Checkpoint result format parameter",
+              "href": "/docs/cloud/checkpoints/manage_checkpoints#configure_the_checkpoint_result_format_parameter"
```

Comment:
```suggestion
          "href": "/docs/cloud/checkpoints/manage_checkpoints#configure-the-checkpoint-result-format-parameter"
```
Docusaurus uses `-` rather than `_` to separate words in anchor links.